packages/vite/src/index.ts

```diff
@@ -1,14 +1,47 @@
 import { transform } from "@kuma-ui/babel-plugin";
 import { Plugin } from "vite";
 import path from "path";
+import {buildSync} from "esbuild";
+import _eval from "eval";
 import { theme, sheet } from "@kuma-ui/sheet";
+import { readdirSync } from "fs";
 
 export type VitePluginOption = {
   breakpoints?: Record<string, string>; // {sm: '400px', md: '700px'}
 };
 
 export default function kumaUI(options?: VitePluginOption): Plugin {
   let mode: "build" | "serve";
+
+  const dir = readdirSync(".");
+  let configPath: string | undefined;
+  dir.forEach((filePath) => {
+    if (filePath.startsWith("kuma.config.")) configPath = filePath;
+  });
+
+  if (configPath) {
+    const filename = path.join(process.cwd(), configPath);
+    const result = buildSync({
+      bundle: true,
+      target: "es2017",
+      write: false,
+      platform: "node",
+      format: typeof require !== 'undefined' ? 'cjs' : 'esm',
+      absWorkingDir: process.cwd(),
+      outfile: filename + ".out",
```

Comment:
Is the outfile actually created? If it is created, it will need to be deleted.

Reply:
The 'write option' is set to false, so it shouldn't write to the file.